packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx

```diff
@@ -265,21 +274,31 @@ export const RichTextSendBox = (props: RichTextSendBoxProps): JSX.Element => {
   ]);
 
   const onRenderSendIcon = useCallback(
-    (isHover: boolean) => (
-      <Icon
-        iconName={isHover && contentValue ? 'SendBoxSendHovered' : 'SendBoxSend'}
-        className={sendIconStyle({
-          theme,
-          hasText: !!contentValue,
-          /* @conditional-compile-remove(attachment-upload) */
-          hasAttachment: false,
-          hasErrorMessage: hasErrorMessage,
-          defaultTextColor: theme.palette.neutralSecondary,
-          disabled: disabled
-        })}
-      />
-    ),
-    [contentValue, disabled, hasErrorMessage, theme]
+    (isHover: boolean) => {
+      // get plain text content from the editor to check if the message is empty
+      // as the content may contain tags even when the content is empty
+      const plainTextContent = editorComponentRef.current?.getPlainContent();
+      const hasContent = !isContentEmpty({
+        plainTextContent: plainTextContent,
+        content: contentValue,
+        placeholder: strings.placeholderText
+      });
+      return (
+        <Icon
+          iconName={isHover && hasContent ? 'SendBoxSendHovered' : 'SendBoxSend'}
+          className={sendIconStyle({
+            theme,
+            hasText: hasContent,
+            /* @conditional-compile-remove(attachment-upload) */
+            hasAttachment: false,
+            hasErrorMessage: hasErrorMessage,
+            defaultTextColor: theme.palette.neutralSecondary,
+            disabled: disabled
```

Comment:
Should we also check whether the content is empty in this line?

Reply:
No, `disabled` will fully disable the send box and it's something that is being set by Contoso. Please see `RichTextSendBoxProps` for more details